src/components/v5/common/ActionSidebar/partials/Motions/steps/StakingStep/StakingStep.tsx

```diff
@@ -32,6 +35,7 @@ const StakingStep: FC<StakingStepProps> = ({ className, isActive }) => {
   const { motionData, colony, token } = motionAction;
   const { usersStakes, motionStakes, requiredStake } = motionData;
 
+  const canInteract = !!wallet && !!user;
```

Comment:
This seems used in a few places around, maybe you could export it from app context?